src/main/scala/beam/utils/plan/PlansBuilder.scala

```diff
@@ -43,8 +43,7 @@ object PlansBuilder {
   val newHH: HouseholdsImpl = new HouseholdsImpl()
   val newHHAttributes: ObjectAttributes = newHH.getHouseholdAttributes
 
-  val modeAllocator: AvailableModeUtils.AllowAllModes =
-    new AvailableModeUtils.AllowAllModes
+  val modeAllocator: AvailableModeUtils.AllowAllModes.type = AvailableModeUtils.AllowAllModes
```

Comment:
If this is truly a static class (functions don't rely on internal state), which it appears is the case, then the standard is to reference it directly. This technically works, but should it be in a variable or just imported?

    import AvailableModeUtils.AllowAllModes

then it can just be referenced:

    AllowAllModes.METHOD

Reply:
Actually this is not a method. is an `Object` inside another `Object`. that is why the word `type` is needed

Reply:
fixed